Alive/Controllers/Alive.php

```diff
@@ -133,6 +143,38 @@ private function assertCacheStatus() :void {
 
 	}
 
+	private function assertBridgeStatus() :void {
+
+		// if test requested in the configuration file
+		if( Config::get('alive', 'assert_bridge_status') !== null && Config::get('alive', 'assert_bridge_status') == 1 ) {
+			$Request_query = new Query;
+			$Request_query->query( 
+				'SELECT "_id_site", "'. Config::get('alive','bridge_expected_column') .'" '.
+				'FROM "'. Config::get('alive','bridge_table') .'" WHERE "_id_site" = '.
+				Config::get('alive','bridge_id') 
+			);
+			$response = Bridge::execute( $Request_query );
+			// if test failed as a crash
+			if( is_null($response) ) {
+				Throw new Exception("Can't access the Filemaker database");
+			}
+			// if an error is found in the request, return it
+			elseif( isset($response['error'] ) ) {
+				Throw new Exception( $response['error'] );
+
+			} 
+			else {
```

Comment:
devrait être un `elseif` (un niveau de [complexité conditionnelle](https://resources.infosecinstitute.com/conditional-complexity-of-risk-models/) gagné)